server/datastore/mysql/migrations/tables/20231107130934_AlterWindowsProfilesUseUuid.go

```diff
@@ -0,0 +1,89 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20231107130934, Down_20231107130934)
+}
+
+func Up_20231107130934(tx *sql.Tx) error {
+	// add the profile_uuid column to the profiles table, keeping the old id.
+	// Note that we cannot set the default to uuid() as functions cannot be used
+	// as defaults in mysql 5.7. It will have to be generated in code when
+	// inserting. Cannot be set as primary key yet as it may have duplicates until
+	// we generate the uuids.
+	_, err := tx.Exec(`
+ALTER TABLE mdm_windows_configuration_profiles
```

Comment:
Since the CREATE TABLE migrations had already been merged to `main`, I didn't want to take any chance and went with the ALTER migration instead of modifying the existing migration.

Reply:
thought about the same thing when we were discussing. A downside of not having a feature branch